docs/src/components/Network.js

```diff
@@ -64,40 +67,178 @@ const getProtocolFee = async (provider, unlockAddress) => {
+            )
+          }),
+        }
+      }),
     {
       token: {
         symbol: nativeCurrency.symbol,
         decimals: nativeCurrency.decimals,
       },
-      balance: await provider.getBalance(ownerAddress),
+      balance: await provider.getBalance(ownerAddress).catch(() => 0),
     },
   ])
   return balances.filter(({ balance }) => balance > 0)
 }
 
-const BurnableTokens = ({ network }) => {
-  const provider = new JsonRpcProvider(
-    network.publicProvider,
-    network.chainId,
-    {
-      batchMaxCount: 10,
+const BurnableToken = ({ network, token, balance, reload }) => {
+  const [hash, setHash] = useState('')
+  const [burning, setBurning] = useState(false)
+  const { authenticated, login } = usePrivy()
+  const { wallets } = useWallets()
+
+  const burn = async () => {
+    if (!authenticated) {
+      login()
+      setBurning(true)
+    } else {
+      // Get a signer
+      const provider = new BrowserProvider(
+        await wallets[0].getEthereumProvider()
+      )
+      const signer = await provider.getSigner()
+
+      // switch network
+      // await wallets[0].switchChain(network.id)
+      await provider.send('wallet_switchEthereumChain', [
+        {
+          chainId: `0x${network.id.toString(16)}`,
+        },
+      ])
+
+      // Send tx
+      const unlock = new Contract(
+        network.unlockAddress,
+        [
+          {
+            inputs: [
+              { internalType: 'address', name: 'token', type: 'address' },
+              { internalType: 'uint256', name: 'amount', type: 'uint256' },
+              { internalType: 'uint24', name: 'poolFee', type: 'uint24' },
+            ],
+            name: 'swapAndBurn',
+            outputs: [],
+            stateMutability: 'nonpayable',
+            type: 'function',
+          },
+          {
+            inputs: [],
+            name: 'swapBurnerAddress',
+            outputs: [{ internalType: 'address', name: '', type: 'address' }],
+            stateMutability: 'view',
+            type: 'function',
+          },
+        ],
+        signer
+      )
+
+      const addressOfTokenToBurn = token.address || ethers.ZeroAddress
+      let tx
+
+      if (addressOfTokenToBurn === ethers.ZeroAddress) {
+        try {
+          tx = await unlock.swapAndBurn.populateTransaction(
+            token.address || ethers.ZeroAddress,
+            balance,
+            3000 // No need to enter a pool fee for native currency
+          )
+        } catch (e) {
+          console.error(e)
+        }
+      } else {
+        const fees = [500, 3000, 10000]
+        for (let fee of fees) {
+          if (!tx) {
+            try {
+              tx = await unlock.swapAndBurn.populateTransaction(
+                token.address || ethers.ZeroAddress,
+                balance,
+                fee // pool for the token, not for UDT/UP
+              )
+            } catch (e) {
+              console.error(e)
+            }
```

Comment:
here you can maybe log only if there is no pool at all, also log the pool fee/token that fails. 

also if a pool exist for two or three fees, will that lead to multiple tx being populated ?